### PR TITLE
Feature/add other users to friends list

### PR DESCRIPTION
**Pull request by amh4**
Pull request for seed database setup

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "react-dom": "^18.2.0",
     "react-router": "^6.3.0",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
```
Comment (josh-p-git) [inline frontend/package.json:13]: why has this been downgraded?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/alexHoggett/acebook-mern-fire/issues"
+  },
+  "homepage": "https://github.com/alexHoggett/acebook-mern-fire#readme",
+  "dependencies": {
+    "jest": "^29.4.2"
+  }
+}
```
Comment (josh-p-git) [inline package.json]: what is this???

**Review by josh-p-git — COMMENTED** [review]
just a couple of questions pls

File: frontend/package.json (hunk 1 above)
Comment (amh4) [reply frontend/package.json:13]: Tbh not sure. This was all either already in there or automatically done.

File: package.json (hunk 2 above)
Comment (amh4) [reply package.json]: The most up to date version of jest I think. I think this is also automatically added

File: frontend/package.json (hunk 1 above)
Comment (josh-p-git) [reply frontend/package.json:13]: i appreciate your honesty

File: package.json (hunk 2 above)
Comment (josh-p-git) [reply package.json]: this thank you message as been automatically generated

**Review by alexHoggett — APPROVED** [review]

**Review by josh-p-git — APPROVED** [review]

**Pull request merged**